src/Common/src/Common.Expression/Spring/Standard/Tokenizer.cs

```diff
@@ -688,4 +688,3 @@ private void RaiseParseException(int start, SpelMessage msg, params object[] ins
         throw new InternalParseException(new SpelParseException(_expressionString, start, msg, inserts));
     }
 }
-#pragma warning restore S125 // Sections of code should not be commented out
```

Comment:
What about the line (6) that starts the suppression?

Reply:
I suspect I've answered this in the other responses. Did you mean something else?

Reply:
Just another instance of the same question. Feels like having something open and not close, but if there's no side affect then I'm not worried about it

Reply:
I get your point. With preprocessor directives its quite common to not 'close' them, in contrast to most other things. For example, consider `#define USE_TRACING` at the top of a file, which makes it easy to turn on/off during debug. If we were to 'close' that one, we'd have to undefine USE_TRACING at the end of the file, which nobody ever does.